.circleci/utils.py

```diff
@@ -0,0 +1,8 @@
+import subprocess
+
+
+def build_and_push_docker(dir, container, tag):
+    build_command = ['docker', 'build', '-t', '%s:%s' % (container, tag), dir]
```

Comment:
Minor: I would use `--tag` instead of `-t`. When you’re in scripts, the extra characters are cheap but can make it easier to understand later.